modules/machineset-creating.adoc

```diff
@@ -19,6 +24,10 @@ specific workloads of your choice.
 * Deploy an {product-title} cluster.
 * Install the OpenShift CLI (`oc`).
 * Log in to `oc` as a user with `cluster-admin` permission.
+ifdef::vsphere[]
+* Create a tag inside your vCenter instance based on the cluster API name. These tags are utilized by the MachineSet to associate the {product-title} nodes to the provisioned virtual machines (VM). For directions on creating tags in vCenter, see link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html[vSphere Tags and Attributes].
```

Comment:
Not critical but just wondering if "These tags" should be "This tag" because the first sentence just says to create a single tag?

Reply:
Also, maybe consider adding "... in the VMware vSphere documentation." or something like that after the link to be clear that you're leaving Red Hat docs?